domain/cloud/state/state.go

```diff
@@ -898,8 +899,8 @@ ON CONFLICT(cloud_uuid, name) DO UPDATE SET name=excluded.name,
 
 // insertPermission inserts a permission for the owner of the cloud during
 // upsertCloud.
-func insertPermission(ctx context.Context, tx *sqlair.TX, ownerName, cloudName string) error {
-	if ownerName == "" {
+func insertPermission(ctx context.Context, tx *sqlair.TX, ownerName user.Name, cloudName string) error {
+	if ownerName.Name() == "" {
```

Comment:
Surely this is `ownerName.IsZero()`?

Reply:
Good point, have done a grep and fixed all the other places this slipped though.